pkg/networkmanager/interfaces.js

```diff
@@ -2576,9 +2856,23 @@ PageNetworkInterface.prototype = {
         var is_deletable = (iface && !dev) || (dev && (dev.DeviceType == 'bond' ||
                                                        dev.DeviceType == 'team' ||
                                                        dev.DeviceType == 'vlan' ||
-                                                       dev.DeviceType == 'bridge'));
+                                                       dev.DeviceType == 'bridge' ||
+                                                       dev.DeviceType == 'wifi'));
         $('#network-interface-delete').prop('hidden', !is_deletable || !managed);
 
+        function render_interface_section_separator(title) {
+            return $('<tr>').append($('<td class="network-interface-separator" colspan="100%">').text(title));
```

Comment:
Can you please use Web API instead of jQuery? And please use class and css rather than `colspan` directly here.

Reply:
Why don't we want to use jQuery here? I see it being used in a few more places along the document, is it a problem with readability?

Reply:
We want to reduce amount of jquery we use, so ideally we don't want to write a new code in it.

Reply:
Okay, I've been able to take jquery out of the equation but since this section is formatted as a table, there is no way for me to take the "colspan" to CSSsince colspan is a structural element in HTML and not a style element and there is no implementation for this in CSS.
Would you rather have the colspan stay, or format the page with divs and flex instead? Or is there another way that I'm not seeing? Thanks!

Reply:
When you use web API, then `setAttribute` should be usable for setting up colspan.
 https://developer.mozilla.org/en-US/docs/Web/API/Element/setAttribute